arquillian-chameleon-deployments/arquillian-chameleon-gradle-build-deployment/pom.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-container-chameleon-parent</artifactId>
+    <groupId>org.arquillian.container</groupId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-chameleon-gradle-build-deployment</artifactId>
+
+  <name>Arquillian Chameleon Gradle Build Deployment</name>
+
+  <properties>
+  	<version.gradle-tooling-api>4.0.1</version.gradle-tooling-api>
+  </properties>
+
+  <repositories>
+  	<repository>
+  	  <id>gradle</id>
+  	  <name>gradle</name>
+  	  <url>http://repo.gradle.org/gradle/libs-releases-local</url>
+  	</repository>
+  </repositories>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.arquillian.container</groupId>
+      <artifactId>arquillian-container-test-spi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.arquillian.container</groupId>
+      <artifactId>arquillian-chameleon-deployment-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+	    <groupId>org.gradle</groupId>
+	    <artifactId>gradle-tooling-api</artifactId>
+	    <version>${version.gradle-tooling-api}</version>
+    </dependency>
+    <dependency>
+    	<groupId>org.jboss.shrinkwrap</groupId>
+	    <artifactId>shrinkwrap-api</artifactId>
+	    <version>1.2.6</version>
+	</dependency>
+    <dependency>
+    	<groupId>org.jboss.shrinkwrap</groupId>
+	    <artifactId>shrinkwrap-impl-base</artifactId>
+	    <version>1.2.6</version>
+	</dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
```

Comment:
I think you are not using mockito anywhere so you can remove this dep.

Reply:
true. removing it.